blocks/footer/footer.css

```diff
@@ -323,7 +323,7 @@ html[dir="rtl"] .ft-main-item:not(:last-child) {
   }
     
   .ft-main .ft-main-item:last-child ul li a {
-    font-size: var(--size-14);
+    font-size: 14px;
```

Comment:
dont we need these variables anymore ?

  --size-18: 18px;
  --size-16: 16px;
  --size-14: 14px;
  --size-12: 12px; etc...